test/byok_test.go

```diff
@@ -7,8 +7,8 @@ func (s *CLITestSuite) TestByok() {
 		// list tests
 		{args: "byok list", fixture: "byok/list_1.golden", login: "cloud"},
 		{args: "byok list --state IN_USE", fixture: "byok/list_2.golden"},
-		{args: "byok list --provider AWS", fixture: "byok/list_3.golden"},
-		{args: "byok list --state IN_USE --provider Azure", fixture: "byok/list_4.golden"},
+		{args: "byok list --cloud AWS", fixture: "byok/list_3.golden"},
+		{args: "byok list --state IN_USE --cloud Azure", fixture: "byok/list_4.golden"},
```

Comment:
Unrelated to your changes, but do you mind using lower-kebab-case for these enums ("IN_USE" --> "in-use", "AWS" --> "aws")? That's the formatting we expect our users to type, and we want to test that they are uppercased correctly before they're passed to the backend.

Reply:
done